cli/viam/main.go

```diff
@@ -251,176 +256,150 @@ func main() {
-								Required: false,
-								Usage:    "locations filter",
+								Name:  dataFlagLocationIDs,
+								Usage: "locations filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagRobotID,
-								Required: false,
-								Usage:    "robot id filter",
+								Name:  dataFlagRobotID,
+								Usage: "robot id filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagPartID,
-								Required: false,
-								Usage:    "part id filter",
+								Name:  dataFlagPartID,
+								Usage: "part id filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagRobotName,
-								Required: false,
-								Usage:    "robot name filter",
+								Name:  dataFlagRobotName,
+								Usage: "robot name filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagPartName,
-								Required: false,
-								Usage:    "part name filter",
+								Name:  dataFlagPartName,
+								Usage: "part name filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagComponentType,
-								Required: false,
-								Usage:    "component type filter",
+								Name:  dataFlagComponentType,
+								Usage: "component type filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagComponentName,
-								Required: false,
-								Usage:    "component name filter",
+								Name:  dataFlagComponentName,
+								Usage: "component name filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagMethod,
-								Required: false,
-								Usage:    "method filter",
+								Name:  dataFlagMethod,
+								Usage: "method filter",
 							},
 							&cli.StringSliceFlag{
-								Name:     dataFlagMimeTypes,
-								Required: false,
-								Usage:    "mime types filter",
+								Name:  dataFlagMimeTypes,
+								Usage: "mime types filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagStart,
-								Required: false,
-								Usage:    "ISO-8601 timestamp indicating the start of the interval filter",
+								Name:  dataFlagStart,
+								Usage: "ISO-8601 timestamp indicating the start of the interval filter",
 							},
 							&cli.StringFlag{
-								Name:     dataFlagEnd,
-								Required: false,
-								Usage:    "ISO-8601 timestamp indicating the end of the interval filter",
+								Name:  dataFlagEnd,
+								Usage: "ISO-8601 timestamp indicating the end of the interval filter",
 							},
 						},
 						Action: DeleteCommand,
 					},
 				},
 			},
 			{
-				Name:  "robots",
-				Usage: "work with robots",
+				Name:            "robots",
+				Usage:           "work with robots",
+				HideHelpCommand: true,
 				Subcommands: []*cli.Command{
 					{
 						Name:  "list",
-						Usage: "list robots",
+						Usage: "list robots in an organization and location",
 						Flags: []cli.Flag{
 							&cli.StringFlag{
-								Name: "organization",
+								Name:        "organization",
+								DefaultText: "currently selected organization",
```

Comment:
I see `currently selected` in a lot of places - how do we select? and is there a good way to surface how to select an org/location?

Reply:
> how do we select?

Good call out; `currently selected` was misleading. It's really the first organization/location alphabetically; updated `DefaultText` here and elsewhere.

> and is there a good way to surface how to select an org/location?

Yeah `select` was a misnomer here; there is no way to "select" an organization apart from using a `--organization` option or relying on the first alphabetical default.